docs/architecture/constant-currency/api.md

```
@@ -0,0 +1,347 @@
+# Constant Currency APIs
+
+Public Cost Management APIs and the internal Masu inspection endpoint for
+constant currency. Base path unless noted:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)